src/store/constants.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export const MODULE_NS = "scaifeWidgets";
```

Comment:
I'm sure you already probably have it on your `TODO` list but we need to make sure to export this from the package too so other modules can access the 'namespaced-state-space'.